src/main/java/yevhenii/lostfilmdemo/services/impl/TVSeriesServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package yevhenii.lostfilmdemo.services.impl;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import yevhenii.lostfilmdemo.entity.TVSeries;
+import yevhenii.lostfilmdemo.services.TVSeriesService;
+
+@Service
+class TVSeriesServiceImpl implements TVSeriesService {
+
+    @Transactional
+    @Override
+    public void add(TVSeries tvSeries) {
+
+    }
+
+    @Override
+    public TVSeries findByName(String name) {
```

Comment:
find by name can return null in case if such episode does not exists, which is a valid case, so in order to indicate that return Optional<TVSeries>